### PR TITLE
ON-12440 # Remove string value type validation from properties

### PR DESCRIPTION
**Pull request by kizaonline**

### CODE REVIEW

File: OneBlink.SDK/models/FormElement.cs [hunk 1]
```diff
@@ -116,21 +64,7 @@ public string type
         public Boolean? isInteger {get; set; }
         public Boolean? displayAsCurrency {get;set;}
         private string _StorageType;
```
Comment (mymattcarroll) [inline OneBlink.SDK/models/FormElement.cs:66]: Can this be removed?

File: OneBlink.SDK/models/FormsAppMenuItems.cs [hunk 2]
```diff
@@ -6,36 +6,10 @@ namespace OneBlink.SDK.Model
...
-            "DRAFTS",
-            "PENDING_SUBMISSIONS",
-            "PROFILE",
-            "HREF",
-            "CONTAINER",
-            "FORM"
-        };
         public string label { get; set; }
         public string icon { get; set; }
         private string _Type;
```
Comment (mymattcarroll) [inline OneBlink.SDK/models/FormsAppMenuItems.cs:11]: Can this be removed?

**Comment by mymattcarroll** [comment]
👍  LGTM

File: OneBlink.SDK/models/FormElement.cs (hunk 1 above)
Comment (kizaonline) [reply OneBlink.SDK/models/FormElement.cs]: 👍

File: OneBlink.SDK/models/FormsAppMenuItems.cs (hunk 2 above)
Comment (kizaonline) [reply OneBlink.SDK/models/FormsAppMenuItems.cs]: 👍

**Pull request merged**